samples/net/sockets/socketpair/src/socketpair_example.c

```diff
@@ -47,7 +47,12 @@ static const char *const names[] = {
 static void hello(int fd, const char *name)
 {
 	/* write(2) should be used after #25443 */
-	send(fd, name, strlen(name), 0);
+	int res = send(fd, name, strlen(name), 0);
```

Comment:
This checks the return value as Coverity wants, but it's not a good example IMO because surely the failure should be returned to the caller and checked there.

Not blocking, net maintainers should weigh in.

Reply:
Since it's sample code, I'm mostly impartial here. I just thought it would might be useful to print an error message to notify the user who would, presumably, be watching.